docs/src/man/getting_started.md

```diff
@@ -423,6 +434,18 @@ julia> df[(df.A .> 500) .& (300 .< df.C .< 400), :]
 │ 98  │ 795   │ 8     │ 398   │
 │ 99  │ 797   │ 8     │ 399   │
 ```
+Where a specific subset of values needs to be matched, the `in()` function can be applied:
+
+```jldoctest dataframe
+julia> df[in([1, 5, 601]).(df.A), :]
```

Comment:
Maybe we should recommend `in.(df.A, Ref([1, 5, 601]))` instead? I find the syntax where `in(...)` is broadcasted quite hard to understand, since it requires knowing that `in(...)` returns an anonymous function (which is an exception). On the contrary, explaining how `Ref` works would be more generally useful.

Reply:
I support the suggestion of @nalimilan and would show both options. Alternatively I can add it in  #1969 after this is merged if you prefer.

Reply:
Have reverted to a call without `Fix2` to introduce the example and mentioned `Fix2` afterwards - it is less than ideal that `Fix2` isn't really documented anywhere, the only mention in the docs I could find was the linked `isequal` example.

Reply:
I have proposed a link that I think is a bit better in this case.

Reply:
Yeah that makes sense - although I still wish there was some actual documentation on `Fix2` somewhere in the docs...